pages/contact-us.html

```diff
@@ -5,6 +5,7 @@
 banner_file_mobile: banner--contact-us-sm.svg
 permalink: /contact-us
 social_image: social-card--contact-us.png
+social_image_alt_text: "Navy blue graphic block with a blue satellite and yellow moon"
```

Comment:
Suggested text edit:

"Graphic illustration of a large satellite dish listening for signals. A yellow moon hangs above the dish in a night time sky."